src/rules/hindi.toml

```diff
@@ -0,0 +1,19 @@
+min_trimmed_length = 0
+min_word_count = 0
+max_word_count = 50
+min_characters = 0
+may_end_with_colon = false
+quote_start_with_letter = false
+needs_punctuation_end = false
+needs_letter_start = false
+needs_uppercase_start = false
+matching_symbols = [
+  ["।", "."]
+]
```

Comment:
in that case `matching_symbols` should be removed. Check the README for an example on what it can be used for.

Reply:
OK